telemetry/opentelemetry-config/src/main/java/io/helidon/telemetry/otelconfig/SpanExporterConfigBlueprint.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.telemetry.otelconfig;
+
+import io.helidon.builder.api.Option;
+import io.helidon.builder.api.Prototype;
+
+@Prototype.Configured
+@Prototype.Blueprint
+@Prototype.CustomMethods(SpanExporterConfigSupport.CustomMethods.class)
+interface SpanExporterConfigBlueprint {
+
+    /**
+     * Span exporter type.
+     *
+     * @return exporter type
+     */
+    @Option.Configured
+    @Option.Default("DEFAULT")
+    ExporterType type();
```

Comment:
The `ExporterType` enum is not declared as `public`, yet it will be part of public API.
I think this is valid for all the enums in this PR - please make sure they are `public` when used in  any `Blueprint`.

Reply:
Unfortunately, I think I have to change all the enums to  `public`.

We _should_ be able to keep them package private. They are used only in blueprints that themselves should not be exposed to developers, and I had marked (most) of the related methods also with access "". 

But the generated getter method on the `BuilderBase` for the enum is public even though the generated setter uses the specified access.

See https://github.com/helidon-io/helidon/issues/10552